index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description" content="This is portfolio repository" />
+    <meta
+      name="author"
+      content="Bhaskar Ranjan Bora, B.Tech Computer Science"
+    />
+    <title>Bhaskar Ranjan Bora</title>
+
+    <link href="css/bootstrap.min.css" rel="stylesheet" />
+    <link href="css/main.css" rel="stylesheet" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  </head>
+  <body>
+    <main>
+        <header>
+            <nav class="navbar navbar-expand-md" aria-label="Fourth navbar example">
+                <div class="container-fluid">
+                <a class="navbar-brand" href="#"><img src="https://dummyimage.com/150X80"></img></a>
+                <button
+                    class="navbar-toggler"
+                    type="button"
+                    data-bs-toggle="collapse"
+                    data-bs-target="#navbarsExample04"
+                    aria-controls="navbarsExample04"
+                    aria-expanded="false"
+                    aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+
+                <div class="collapse navbar-collapse" id="navbarsExample04">
+                    <ul class="navbar-nav me-auto mb-2 mb-md-0">
+                    <li class="nav-item">
+                        <a class="nav-link" href="#portfolio">Portfolio</a>
+                    </li>
+
+                    <li class="nav-item">
+                        <a class="nav-link" href="#about">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#Portfolio">Portfolio</a>
+                    </li>
+                    </ul>
+                </div>
+                </div>
+            </nav>
+        </header>
+        <section class="section1">
+            <div class="hero-section">
+                <div class="child">
+                    <div class ="title"><h1>I am Bhaskar Ranjan Bora, Glad to see you here.</h1></div>
+                    <div class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                    </div>
+
+                    <div class="icons">
+                            <li><i class="fa fa-facebook fa-2x"></i></li>
+                            <li><i class="fa fa-twitter fa-2x"></i></li>
+                            <li><i class="fa fa-instagram fa-2x"></i></li>
```

Comment:
- [ ] Kindly wrap your `li` tags in the `ul`` tag because every time it must be wrapped in the `li` tag must be wrapped into the `ul` tag. :wink: 
- [ ] Kindly wrap your social media icons in the `a` tag which will increase user experience. :wink: